server/types/src/components.rs

```diff
@@ -104,3 +104,16 @@ impl MessageReceiver {
         self.buffer.drain(..)
     }
 }
+
+/// Component which stores a number which used for OpenWindow packets, incremented on every access
+#[derive(Default, Debug)]
+pub struct OpenWindowCount {
```

Comment:
I believe the counter is supposed to start at 1, according to the wiki page? `u8::default()` returns 0

Reply:
I saw no issues with it starting at 0 but I can change it to follow the wiki.